cmd/mock-es/main.go

```diff
@@ -47,6 +50,7 @@ func init() {
 	flag.StringVar(&certFile, "certfile", "", "path to PEM certificate file, empty sting is no TLS")
 	flag.StringVar(&keyFile, "keyfile", "", "path to PEM private key file, empty sting is no TLS")
 	flag.DurationVar(&delay, "delay", 0, "Go 'time.Duration' to wait before processing API request, 0 is no delay")
+	flag.BoolVar(&logRequests, "logrequests", false, "Enable request logging")
```

Comment:
Proposal:  How about changing this to "verbose", and then both the "starting HTTP server" and the body logging could be enabled or disabled with that flag?

I like being able to keep the server silent, or silent with metrics I can pipe to jq.

Reply:
That makes more sense, I'll update the PR